lib/docusign_transaction_rooms/version.rb

```diff
@@ -1,3 +1,3 @@
 module DocusignTransactionRooms
-  VERSION = "0.1.7"
+  VERSION = "0.1.8"
```

Comment:
I would argue this would be a "0.2.0" It's not really a bug fix so much as adding pagination as a feature.

Reply:
Good call